src/organizations/organizations.service.ts

```diff
@@ -1,26 +1,64 @@
-import { Injectable } from '@nestjs/common';
+import {
+  ConflictException,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
 import { CreateOrganizationDto } from './dto/create-organization.dto';
 import { UpdateOrganizationDto } from './dto/update-organization.dto';
+import { PrismaService } from 'src/prisma/prisma.service';
+import * as bcrypt from 'bcryptjs';
+import { ConfigService } from '@nestjs/config';
 
 @Injectable()
 export class OrganizationsService {
-  create(createOrganizationDto: CreateOrganizationDto) {
-    return 'This action adds a new organization';
+  constructor(
+    private configService: ConfigService,
+    private readonly prisma: PrismaService,
+  ) {}
+
+  private salt = this.configService.getOrThrow('SALT');
```

Comment:
if your do the validation in module then you wont need to throw anything here. If an dependency that will make your code break can be detected when the app is starting, then there is no point running it at all. Inform the user at that point to provide all those items first.